app/assets/stylesheets/custom.css.scss

```diff
@@ -895,6 +895,18 @@ $grayMediumLight: #eaeaea;
   /*float: right;*/
 }
 
+.rounded-box {
+  position: relative;
```

Comment:
Properties should be ordered border, -moz-border-radius, -webkit-border-radius, border-radius, padding, position